context/app/static/js/pages/CellsAPIDemo/CellsService.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable class-methods-use-this */
+// Right now, the object doesn't have any state, but down the road,
+// it might cache results, or the API calls might be made from the JS instead of Python,
+// and then the instance would be initialized with the Cells API URL.
+
+class CellsService {
+  async fetchAndParse(url) {
```

Comment:
Conceivably, the whole client could be moved from Python to JS... but for now, I think the Python responses will have a consistent, simple structure that this will handle.